bambam.py

```diff
@@ -69,6 +71,12 @@ def load_image(cls, fullname, colorkey=None):
         except pygame.error as message:
             print("Cannot load image:", fullname)
             raise SystemExit(message)
+
+        sz_x, sz_y = image.get_rect().size
+        if (sz_x > cls.IMAGE_MAX_WIDTH or sz_y > cls.IMAGE_MAX_WIDTH):
+            new_size = (cls.IMAGE_MAX_WIDTH, int(cls.IMAGE_MAX_WIDTH * (float(sz_y)/sz_x)))
```

Comment:
Could you please raise an exception here if `new_size[1]` is less than one? Please include `fullname` in the message.